sdk/src/inflation.rs

```diff
@@ -22,7 +22,7 @@ pub struct Inflation {
     __unused: f64,
 }
 
-const DEFAULT_INITIAL: f64 = 0.15;
+const DEFAULT_INITIAL: f64 = 0.08;
```

Comment:
Any point in actually changing this?   It would only affect new development clusters, and have no affect for devnet/testnet/mainnet-beta.   That being said, no objections to the change itself.

Reply:
Just to keep alignment with design but I'm fine with waiting if it would lead to confusion now. I defer to you

Reply:
sg, thanks.  Changing it wfm then